app/assets/javascripts/short-form/FileUploadService.js.coffee

```diff
@@ -1,7 +1,8 @@
 FileUploadService = ($http, Upload, uuid, ShortFormApplicationService) ->
   Service = {}
-  Service.session_uid = ShortFormApplicationService.session_uid
   Service.preferences = ShortFormApplicationService.preferences
+  Service.session_uid = ->
```

Comment:
@chrisdolendo yep! because I added an update so that `session_uid` can get refreshed every time you start a new short form app (stops you from accidentally picking up a file upload from a short form session in your same browser window, if you start a new application). So if the value is an assignment, then FileUploadService.session_uid will always be assigned to the initial value. If it's a function then it will run the function, and pick up whatever is the _current_ value in SFApp service.